test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityPCLTests.cs

```diff
@@ -32,6 +32,115 @@ public void CompatibilityPCL_Net()
             Assert.False(compat.IsCompatible(framework2, framework1));
         }
 
+        [Fact]
+        public void CompatibilityPCL_WithOneTfm()
+        {
+            var framework1 = NuGetFramework.Parse("net45");
+            var framework2 = NuGetFramework.Parse("portable-net45");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.True(compat.IsCompatible(framework1, framework2));
+            Assert.False(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Fact]
+        public void CompatibilityPCL_ZeroTfmIntoNonPclProject()
+        {
+            var framework1 = NuGetFramework.Parse("net45");
+            var framework2 = NuGetFramework.Parse("portable");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.False(compat.IsCompatible(framework1, framework2));
+            Assert.False(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Fact]
+        public void CompatibilityPCL_ZeroTfm()
+        {
+            var framework1 = NuGetFramework.Parse("portable");
+            var framework2 = NuGetFramework.Parse("portable");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.True(compat.IsCompatible(framework1, framework2));
+            Assert.True(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Theory]
+        // supported profiles
+        [InlineData("portable-net45+netcore45", "netstandard6.0", false)]          // Profile7 -> netstandard 1.0 - 1.1
```

Comment:
I re-read that and it is pretty confusing. The intent was to say that PCL projects with a specific profile number can directly be mapped to a netstandard version unambiguously, and thus can use netstandard packages. Packages that have portable-x+y monikers sometimes cannot be mapped directly to a netstandard version because the profiles the TFMs that come after + may not be in that profile.

In other words, disallow making packages that have portable-x+y -> netstandard, but allow portable projects to pull in netstandard packages. That stops the bleeding and allows package authors to make new packages that explicitly support netstandard.